frontend/src/lib/components/TopBar.svelte

```diff
@@ -29,6 +30,8 @@
 </script>
 
 <div class="top-bar">
+	<BranchButton initialBranchName="Set Branch" />
+	<div class="spacer"></div>
```

Comment:
This could probably be done using CSS, without a separate element. If not, no biggie, but it's worth a shot

Reply:
I actually probably don't need it, it is actually set in main.ts which will override this.  I should look into removing this.